ckan/logic/action/delete.py

```diff
@@ -391,7 +396,8 @@ def group_delete(context, data_dict):
 def organization_delete(context, data_dict):
     '''Delete an organization.
 
-    You must be authorized to delete the organization.
+    You must be authorized to delete the organization
+    and no datsets should belong to the organization
```

Comment:
typo in "datasets", plus it should also say "unless ckan.auth.create_unowned_dataset=True"